numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -373,8 +373,7 @@ static int byteorder_parser(char const *str, Py_ssize_t length, void *data)
 
     if (length < 1) {
         return -1;
-    }
-    else if (str[0] == NPY_BIG || str[0] == NPY_LITTLE
+    } else if (str[0] == NPY_BIG || str[0] == NPY_LITTLE
```

Comment:
I don't think we usually use this code style.

Reply:
will do, thanks for pointing. separate discussion: would be nice to have a static analysis tool running style checks for these type of issues in the CI.